### PR TITLE
Cc0h

### PR DESCRIPTION
**Pull request by cc0h**
preview my PR plz！

### CODE REVIEW

**Review by irvingskr — CHANGES_REQUESTED** [review]
Update your repository.

**Comment by cc0h** [comment]
I have fixed some silly mistakes,review it again plz.

File: 2022-finalproject/README.md [hunk 1]
```diff
@@ -1,44 +1,28 @@
...
+   1. git revert
+    
+    ![revert1](revert1.png)
+    ![revert2](revert2.png)
+
+
+## Question3
+
+1. 合并某分支上的单个commit：git cherry-pick 文件名
+2. 在IDEA中合并
```
Comment (irvingskr) [inline 2022-finalproject/README.md:28]: Have you read the task seriously, you should new a directory named by your github id and put all your task file into it.

**Review by irvingskr — CHANGES_REQUESTED** [review]

File: 2022-finalproject/README.md (hunk 1 above)
Comment (cc0h) [reply 2022-finalproject/README.md]: 忘记这点了，抱歉（）

**Comment by cc0h** [comment]
已经把文件都放到文件夹里了

File: 2022-finalproject/cc0h/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+## question1
+
+提交了一个文件
+![pic1](1.png)
+发现文件有误
+使用reset命令（或git commit --amend来重新提交修改过的文件）
+![reset](reset.png)
+### 总结
+
+1. reset撤回修改后重新提交
```
Comment (irvingskr) [inline 2022-finalproject/cc0h/README.md:10]: Just reset?

File: 2022-finalproject/cc0h/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+   1. git revert
+    
+    ![revert1](revert1.png)
+    ![revert2](revert2.png)
+
+
+## Question3
+
+1. 合并某分支上的单个commit：git cherry-pick 文件名
+2. 在IDEA中合并
```
Comment (irvingskr) [inline 2022-finalproject/cc0h/README.md:28]: > **要求：** 编写一个文档，回答下列问题，这些问题的答案都应当使用 git 命令实现  
Please read the task carefully.

**Review by irvingskr — CHANGES_REQUESTED** [review]

File: 2022-finalproject/cc0h/README.md (hunk 3 above)
Comment (cc0h) [reply 2022-finalproject/cc0h/README.md]: 已解决

**Review by irvingskr — APPROVED** [review]

**Review by irvingskr — APPROVED** [review]

**Pull request merged**